library/core/src/time.rs

```diff
@@ -1128,11 +1128,11 @@ impl fmt::Debug for Duration {
                     let s = unsafe { crate::str::from_utf8_unchecked(&buf[..end]) };
 
                     // If the user request a precision > 9, we pad '0's at the end.
-                    let w = f.precision().unwrap_or(pos);
-                    write!(f, ".{:0<width$}", s, width = w)?;
+                    let width = f.precision().unwrap_or(pos);
+                    write!(f, ".{s:0<width$}")?;
```

Comment:
I realize this is legal for the fmt syntax but I would prefer we not.

Reply:
reverted